packages/turf-intersect/test/in/issue-1132-line.geojson

```diff
@@ -0,0 +1,241 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [
+              -8479843.24714559,

```

Comment:
@Knight219 You're coordinates are not using Decimal Degrees.

Coordinates that are not using a WGS84 projection is not going to be supported by TurfJS or the GeoJSON specification.

Could you replace your test data using Decimal Degrees instead.

Reply:
Intresting:). We use turf 4.0.0 with this format and all works well.

Reply:
Humm... 🤔 No clue then, right now the output of your test data is returning a `Null` geometry.

https://travis-ci.org/Turfjs/turf/builds/308889447#L3074-L3076

Regardless, we won't be supporting no-decimal degree coordinates in TurfJS.

Reply:
I'll update our code and tests

Reply:
Looks that my data also in WGS84 with EPSG3857 but not EPSG4236

Reply:
@Knight219 🤔 ? Does it also fail using WGS84? If so then send a commit with the change of test data.

If we don't come up with a solution we can skip it for now but at least the test case will be present for the future.